labonneboite/web/static/js/unobfuscate.js

```diff
@@ -10,10 +10,18 @@
     var elements = this;
     if (elements.length) {
       elements.each(function () {
-        var reversed = $(this).text().split('').reverse().join('');
+        var $this = $(this);
+
+        var reversed = $this.text().split('').reverse().join('');
+
+        var title = $this.attr('data-title') || 'Nous contacter par e-mail';
+
+        var mailSubject = $this.attr('data-subject');
+        if(mailSubject) reversed = reversed + '?subject=' + encodeURIComponent(mailSubject);
+
         $(this)
           .removeClass('js-obfuscated-email')
-          .html('<a href="mailto:'+ reversed +'">Contact</a>');
+          .html('<a title="' + title + '" href="mailto:'+ reversed +'">Contact</a>');
```

Comment:
Je me pose vraiment la question de la pertinence de cette obfuscation. Je fais ça aussi sur mon site perso, mais la google console se plaint que le contenu change durant le rendering, ce qui impacte le SEO. Par ailleurs, je me demande si les crawlers de spam ne font pas du rendering de la page de toute façon, ce qui rend l'obfuscation inutile.

Mais bon, cette remarque ne rentre pas dans le périmètre de cette PR.

Reply:
Personnellement, je l'utilise *just in case*...